src/configuration/app-containers.md

```diff
@@ -40,6 +40,18 @@ The `.platform.app.yaml` file needs at least one of the following to define an i
 
 Each web or worker instance is its own running container, which takes its own resources.  The `size` key allows some control over how many resources each container gets and if omitted the system will select one of a few fixed sizes for each container automatically.  All application and service containers are given resources out of a common pool defined by your plan size.  That means the more containers you define, the fewer resources each one will get and you may need to increase your plan size.
 
+## Compression
+
+Platform.sh does not compress any dynamic responses generated by your application due to a [well known security issue](https://en.wikipedia.org/wiki/BREACH_%28security_exploit%29).  While your application can compress its own response, doing so when the response includes any user-specific information, including a session cookie, opens up an attack vector over SSL connections.  For that reason we recommend against compressing any generated responses.
```

Comment:
`s/SSL/TLS/`?

Reply:
I'll put both.  TLS may be more accurate but most people only know "SSL is that secure thingie that makes my connection safe, right?"